.claude/commands/discover.md

```
@@ -34,7 +34,7 @@ cd "${HOME}/.claude-octopus/plugin" && bash scripts/orchestrate.sh probe <user's
 ```
 
 **✗ INCORRECT:**
-```
+```text
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)